docs/README.md

```diff
@@ -0,0 +1,27 @@
+## Supported Platforms
+iOS and Android You.i Engine Applications can be automated using Appium. Support for other platforms is planned for future releases.
+
+### Xcode Selection
+Automation for iOS 10 is supported with Xcode 8+
+Automation for iOS 9 or lower is supported with Xcode 7.3.1
+
+The reasoning behind the variation in version is due to the change in the underlying automation layer on iOS. As of iOS 10, Apple changed its underlying automation from Instruments to UIAutomation using [XCUITest](https://github.com/appium/appium-xcuitest-driver) appium driver
```

Comment:
As of iOS 10, Apple changed its underlying automation 'in Instruments from UIAutomation to XCUITest'